skimage/draw/__init__.py

```diff
@@ -1,3 +1,51 @@
+"""Utilities for generating coordinates of different shapes and curves.
+
+Use this module for generating coordinates of a bezier curve, circle, ellipse, rectangle, 
+generating ellipsoid, calculating surface-area and volume of ellipsoid, 
+drawing lines, setting pixel color at a coordinate.
```

Comment:
```suggestion
Use the tools in this module to generate coordinates of basic geometric shapes
curves such as ellipses or bezier curves. Additional tools allow the creation
of masks from polygons or setting the color of manipulation.
```

What do you think about this? Note that we are following [PEP8](https://peps.python.org/pep-0008/) in our library, so I wrapped the line length to 79 chars.

Reply:
the way a developer use a docstring will be like, hovering the mouse over the function, most of the time he does this is to just verify whether the function he has chosen to use, will that function be able to do what he intends. my version of docstring seems to be giving that clarity on available functionalities with the exact words, your version seems to me like it won't be suitable for that verification intention, I would like if you could rethink about my perspective on this difference.

Reply:
I may misunderstand, but I think you are arguing that listing all provided shapes is more explicit and clear? I would disagree. Being clear *is* valuable, but I don't think making the user read through a long enumeration is clear at all. He can already get a precise and complete list by using the autocomplete after typing `skimage.draw.` or look it up on our website.  On the other hand, most people know "basic geometric" shapes will know that this includes all the above listed shapes and it's way more precise.

And finally, again the long-term maintenance is something we need to consider here. Each time, we change which shapes which are provided in this module we have to update this list by hand. I can say from experience, that developers will forget to update this.

Reply:
this dynamics that you have mentioned, " considering the long term maintenance ", that's a new perspective to me. the change that you have suggested, I will go with it.